full-service/src/db/transaction_log.rs

```diff
@@ -407,9 +426,7 @@ impl TransactionLogModel for TransactionLog {
     }
 
     fn get(id: &TransactionId, conn: Conn) -> Result<TransactionLog, WalletDbError> {
-        use crate::db::schema::transaction_logs::dsl::{id as dsl_id, transaction_logs};
```

Comment:
I'm not sure if there was a reason the use statements were in each function, it seems to cause a bit of code duplication and detracts from focusing on the function implementations.